src/client/tsconfig.json

```diff
@@ -9,7 +9,7 @@
     "types": ["vite/client", "node", "jest"],
     "allowJs": false,
     "skipLibCheck": true,
-    "esModuleInterop": false,
+    "esModuleInterop": true,
```

Comment:
just need reminder for why we have switched this ... there are consequences, with mitigations, as it will create a lot of boilerplate code in our generated js

Reply:
jest wasn't able to recognise a default import unless this setting was switched on, I will make a note to revisit it after the conversion to vitest